src/contexts/portal-settings/PortalSettingsContext.js

```diff
@@ -1,6 +1,9 @@
 import { createContext } from "react";
 
+const { GATSBY_PORTAL_DOMAIN } = process.env;
+
 export const defaultSettings = {
+  supportEmail: GATSBY_PORTAL_DOMAIN ? `hello@${GATSBY_PORTAL_DOMAIN}` : null,
```

Comment:
Is this a good assumption? I can remove it all together, but I figured this would be a nice touch.

Reply:
That's what we use for emails https://github.com/SkynetLabs/webportal-website-skynetlabs/blob/main/src/services/emails.js so the assumption is good. However the env variable will not be available if you don't assign `PORTAL_DOMAIN` to `GATSBY_PORTAL_DOMAIN` in skynet-webportal repo which is kind of annoying imo so instead I used a plugin to allow gatsby to access those https://github.com/SkynetLabs/webportal-website-skynetlabs/blob/main/gatsby-config.js#L45-L50

Reply:
That's a cool plugin! I'll make it a follow up, since `GATSBY_PORTAL_DOMAIN` is already assigned and used in the dashboard.

* [Reference](https://github.com/SkynetLabs/skynet-webportal/blob/79c41d5979833584b699ddce30f483cc29d18e1a/docker-compose.accounts.yml#L69-L70)
* [Task [SKY-981]](https://linear.app/skynetlabs/issue/SKY-981/use-a-plugin-for-easier-access-to-env-variables)